modules/ROOT/pages/anypoint-mq/3.x/anypoint-mq-connector-reference.adoc

```diff
@@ -0,0 +1,526 @@
+|===
+|Type |Binary
+| Attributes Type a| <<AnypointMQMessageAttributes>>
+|===
+
+==== For Configurations
+* <<config>>
+
+==== Throws
+* ANYPOINT-MQ:ACKING
+* ANYPOINT-MQ:CONSUMING
+* ANYPOINT-MQ:CONNECTIVITY
+* ANYPOINT-MQ:TIMEOUT
+* ANYPOINT-MQ:DESTINATION_NOT_FOUND
+* ANYPOINT-MQ:RETRY_EXHAUSTED
+
+
+[[nack]]
+=== Nack
+`<anypoint-mq:nack>`
+
+
+Executes an Not Acknowledgment and change the status of the message from In Flight to In Queue to be consumed again for a subscriber
+
+
+==== Parameters
+[%header%autowidth.spread]
+|===
+| Name | Type | Description | Default Value | Required
+| Configuration | String | Name of the configuration to use. | | x
+| Ack Token a| String |  The `ackToken` of the message, which was received when message was consumed. `ackToken` will only be created when acknowledgmentMode="MANUAL" is used. |  | x
+| Reconnection Strategy a| * <<reconnect>>
+* <<reconnect-forever>> |  A retry strategy in case of connectivity errors |  |
+|===
+
+
+==== For Configurations
+* <<config>>
+
+==== Throws
+* ANYPOINT-MQ:NACKING
+* ANYPOINT-MQ:CONNECTIVITY
+* ANYPOINT-MQ:DESTINATION_NOT_FOUND
+* ANYPOINT-MQ:RETRY_EXHAUSTED
+
+
+[[publish]]
+=== Publish
+`<anypoint-mq:publish>`
+
+
+==== Parameters
+[%header%autowidth.spread]
+|===
+| Name | Type | Description | Default Value | Required
+| Configuration | String | Name of the configuration to use. | | x
+| Destination a| String |  Queue or Exchange name from which to fetch a Message |  | x
+| Output Mime Type a| String |  The mime type of the operation output. |  |
+| Output Encoding a| String |  The encoding of the operation output. |  |
+| Streaming Strategy a| * <<repeatable-in-memory-stream>>
+* <<repeatable-file-store-stream>>
+* non-repeatable-stream |  Specifies to use repeatable streams. |  |
+| Body a| Binary |  The body of the outgoing MQ Message, sent as a String as part of a JSON message. Any content that is published needs to be serializable to a String format that can be later deserialized from. One way to do so is to serialize the published Object to its JSON representation, which can be serialized/deserialized from a String easily. Sending Objects that don't have a proper string serialization will cause the message to be unreadable on the receiving end. |  #[payload] |
+| User Properties a| Object |  The custom user properties that should be set to this Anypoint MQ Message |  |
+| Message Id a| String |  A custom ID for the outgoing Anypoint MQ Message. This value must be a proper UID for the message to avoid delivery problems. If absent, the broker will create one UUID for each message. |  |
+| Send Content-Type a| Boolean |  Set to true if the body type should be sent as a Anypoint MQ Message property |  true |
+| Target Variable a| String |  Name of a variable to store the operation's output. |  |
+| Target Value a| String |  An expression to evaluate against the operation's output and store the expression outcome in the target variable |  #[payload] |
```

Comment:
operation output

"An expression..." stores "the expression outcome"? Reword to clarify.